### ISSUE TITLE
Create ASAP dataset with argument component tasks

### ISSUE THREAD

**Issue by go-bears** [go-bears/essay-feedback-aiedu#16]
Take ASAP training, test and validation sets and add the component markers with a reasoning LLM, any reasoning LLM okay.

- [ ] test prompt & eyeball that categories look correct
- [ ] training set tagged
- [ ] test set tagged
- [ ] validation set tagged

Stretch: recreate the dataset with a different reasoning model to check for interrater agreement with the LLMs.

**Comment by go-bears** [comment]
@jjordanoc I'll take this issue since Xavier's out for the next week. I run the job from my server or laptop since its just calling the the api.
